CHANGELOG.md

```diff
@@ -8,7 +8,13 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* None.  
+* Set direct and transitive dependency header search paths for pod targets  
+  [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
+  [#7116](https://github.com/CocoaPods/CocoaPods/pull/7116)
+
+* Log target names missing host for libraries  
```

Comment:
@keith I was too fast to merge on your PR https://github.com/CocoaPods/CocoaPods/pull/7346, fixed here as a separate commit

Reply:
sorry! I didn't have any conflicts so I thought we were safe

Reply:
I should've double checked, my bad

Reply:
<3